packages/data-context/src/codegen/spec-options.ts

```diff
@@ -53,18 +55,19 @@ export class SpecOptions {
       throw new Error('Cannot generate a spec without a framework')
     }
 
-    // This only works for Vue projects right now. If the framework is not Vue, we're generating an empty component test
-    if (this.options.framework.codeGenFramework !== 'vue') {
-      return {
-        codeGenType: this.options.codeGenType,
-        fileName: await this.buildFileName(),
-        templateKey: 'componentEmpty' as TemplateKey,
-        overrideCodeGenDir: '',
-      }
+    let frameworkOptions
+
+    switch (this.options.framework.codeGenFramework) {
+      case 'react':
+        frameworkOptions = await this.getReactSpecOptions()
+        break
+      case 'vue':
+        frameworkOptions = await this.getVueSpecOptions()
+        break
+      default:
+        throw new Error(`Unable to generate spec for ${this.options.framework.codeGenFramework}`)
```

Comment:
```suggestion
    switch (this.options.framework.codeGenFramework) {
      case 'react':
        return await this.getReactSpecOptions()
      case 'vue':
        return await this.getVueSpecOptions()
      default:
        throw new Error(`Unable to generate spec for ${this.options.framework.codeGenFramework}`)
```

I think this can be simplified, unless I'm missing something.